### ISSUE TITLE
Tsoha-arviointi

### ISSUE THREAD

**Issue by ranilar** [jaakkoset/latin_word_inflection#1]
I did not manage to launch the project after installing the needed components (also the ones not mentioned)., but I looked through the projects files. The subject is very interesting and unique. I'm excited to see the end result. 
 The database structure seems well thought out and I like that it has been explained with comments.  As of now there is not much to comment about anything else as there seems to be more to be done ahead.
 TLDR; Nice unique idea, good database structure and excited to see how it tursn out!
